frontend/lib/src/components/widgets/DataFrame/hooks/useCustomRenderer.ts

```diff
@@ -98,58 +107,65 @@ type CustomRendererReturn = Pick<
  * a red indicator for required cells.
  *
  * @param columns - The columns of the table.
- * @param hasSelectionRow - Whether the table has the selection row activated (e.g. for dynamic editing).
  *
  * @returns An object containing the following properties:
  * - `drawCell`: A function that overwrites some rendering that can be
  *    passed to the `DataEditor` component.
  * - `customRenderers`: A map of custom cell renderers used by custom cells
  *    that can be passed to the `DataEditor` component.
  */
-function useCustomRenderer(
-  columns: BaseColumn[],
-  hasSelectionRow: boolean
-): CustomRendererReturn {
-  const drawCell: DrawCustomCellCallback = React.useCallback(
-    args => {
+function useCustomRenderer(columns: BaseColumn[]): CustomRendererReturn {
+  const drawCell: DrawCellCallback = React.useCallback(
+    (args, draw) => {
       const { cell, theme, ctx, rect } = args
-      const colPos = hasSelectionRow ? args.col - 1 : args.col
+      const colPos = args.col
       if (isMissingValueCell(cell) && colPos < columns.length) {
-        let contentRendered = false
         const column = columns[colPos]
 
         // We explicitly ignore some cell types here (e.g. checkbox, progress...) since
```

Comment:
Should this comment be updated? Also, do we support checkbox and progress? I don't see that in https://github.com/glideapps/glide-data-grid/pull/810

Reply:
Those are our type names in Streamlit. I think the comment is still accurate.